Simplify logic for determining reduction level in ArchiveTable.

mgdaily opened the pull request:
Curtis keeps finding bugs in things!

From a Slack conversation:

![image](https://user-images.githubusercontent.com/7182533/123001145-4227e300-d365-11eb-903c-28ae043f33ac.png)

I went into the logic and noticed some outdated stuff with respect to reduction levels. Since the portal only reports raw/reduced, I kept that convention. I just updated so that anything not RLEVEL=0 is reduced. Also removed quicklook since that's not a thing anymore.

I couldn't find anything else that referenced quicklook, so I think this is the only place?

I've got this running on http://observation-portal-dev.lco.gtn/

Example request with reduced data: http://observation-portal-dev.lco.gtn/requests/1889951

Also confirmed to work on -92 data: 
![image](https://user-images.githubusercontent.com/7182533/123001722-e1e57100-d365-11eb-87ae-516e9efaf4fb.png)

markBowman commented:
You might get a backlog team point for this - https://www.pivotaltracker.com/story/show/169026690

mgdaily commented:
> You might get a backlog team point for this - https://www.pivotaltracker.com/story/show/169026690

I mean I'll take it, buuuut that's for the archive client 😢 I guess we fixed that last sprint with the new archive client - maybe the sprint before that too when we updated the old one?

markBowman commented:
> > You might get a backlog team point for this - https://www.pivotaltracker.com/story/show/169026690
> 
> I mean I'll take it, buuuut that's for the archive client  I guess we fixed that last sprint with the new archive client - maybe the sprint before that too when we updated the old one?

Feel free to drag the story into the sprint

eheinrich reviewed (approved):
Looks good
